client-v2/src/components/FeedsContainer.tsx

```diff
@@ -26,18 +28,28 @@ import { DEFAULT_PARAMS } from 'services/faciaApi';
 import ScrollContainer from './ScrollContainer';
 import ClipboardHeader from 'components/ClipboardHeader';
 import { media } from 'shared/util/mediaQueries';
+import ContainerHeading from 'shared/components/typography/ContainerHeading';
+import { ClearIcon } from 'shared/components/icons/Icons';
+import Button from 'shared/components/input/ButtonDefault';
+import { selectIsPrefillMode } from 'selectors/feedStateSelectors';
 
 interface FeedsContainerProps {
   fetchLive: (params: object, isResource: boolean) => void;
   fetchPreview: (params: object, isResource: boolean) => void;
+  hidePrefills: () => void;
+  isPrefillMode: boolean;
   liveArticles: CapiArticle[];
   previewArticles: CapiArticle[];
+  prefillArticles: CapiArticle[];
   liveLoading: boolean;
   previewLoading: boolean;
+  prefillLoading: boolean;
   liveError: string | null;
   previewError: string | null;
+  prefillError: string | null;
   livePagination: IPagination | null;
   previewPagination: IPagination | null;
+  prefillPagination: IPagination | null;
 }
```

Comment:
Can we implement this in such a way that our component doesn't have to reason about the source of its data?

(I know that you're simply extending a pattern that's already being used -- this was not ideal before! _But_ it seems like a good opportunity to clean this up 😁)